lnbits/core/crud.py

```diff
@@ -61,25 +61,35 @@ async def get_account(
     return User(**row) if row else None
 
 
-async def get_user(user_id: str, conn: Optional[Connection] = None) -> Optional[User]:
+async def get_user(
+    user_id: str, conn: Optional[Connection] = None, deleted: Optional[bool] = False
+) -> Optional[User]:
     user = await (conn or db).fetchone(
         "SELECT id, email FROM accounts WHERE id = ?", (user_id,)
     )
 
     if user:
+        clauses = ["user = ?"]
+        values: List[Any] = [user_id]
+
+        if not deleted:
```

Comment:
- if `deleted == True` then the clause will not be added and both deleted and non-deleted wallets will be returned

Reply:
Doesn't that change the existing behaviour of the function? Wouldn't we have to track all calls to the function and change to `get_user(....., deleted = False)` ?